docker-compose.yml

```diff
@@ -186,6 +186,16 @@ services:
     networks:
       - internal
   postgres:
+    command:
+      - postgres
+      - -c
+      - shared_preload_libraries=pg_stat_statements
+      - -c
+      - pg_stat_statements.track=all
+      - -c
+      - pg_stat_statements.max = 10000
+      - -c
+      - track_activity_query_size = 2048
```

Comment:
This is a modification on https://github.com/ankane/pghero/blob/master/guides/Query-Stats.md#installation 

> Add the following to your `postgresql.conf`:
>
> ```
> shared_preload_libraries = 'pg_stat_statements'
> pg_stat_statements.track = all
> pg_stat_statements.max = 10000
> track_activity_query_size = 2048
> ```